First_section.html

```diff
@@ -211,29 +359,29 @@ <h2 class="link">LET'S CONNECT</h2>
               </li>
               <li>
                 <a href="#">
-                  <img class="social" src="Vector (1).png" />
+                  <img class="social" src="Vector (2).png" />
                 </a>
               </li>
               <li>
                 <a href="#">
-                  <img class="social" src="Vector (3).png" />
+                  <img class="social" src="Vector.png" />
                 </a>
               </li>
               <li>
                 <a href="#">
-                  <img class="social" src="Vector (2).png" />
+                  <img class="social" src="Vector (1).png" />
                 </a>
               </li>
             </ul>
           </div>
           <ul class="link_to_resume">
             <li class="resume"><a href="#">Get my resume</a></li>
           </ul>
-        </section>
-
+      </section>
+    
         <ul class="skill_set">
            <li class="set">
-            <h1 class="language">Languages</h1>
+            <h3 class="language">Languages</h3>
```

Comment:
- [ ] Kindly make these parts aligned in a column as shown in Figma 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/207922442-019feea8-1ca0-4402-86f4-0d345b78adbe.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/207922507-735c65ab-2195-4baf-89ea-0e39adea51d4.png)

Reply:
Done, thank you for the kind feedback.